testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/ProfileSettingsActivity.kt

```diff
@@ -75,9 +83,20 @@ class ProfileSettingsActivity : BaseActivity() {
             val imageUri = data?.data
             setProfileImage(imageUri)
             profileUrl = imageUri.toString()
+            profileUrlBase64 = encodeImageForMiniApp(profileUrl)
         }
     }
 
+    private fun encodeImageForMiniApp(profileUrl: String): String {
```

Comment:
This function should run on worker thread. For example, you can use `withContext(Dispatchers.IO){}`.